_posts/2019-10-20-iPhone-11.md

```diff
@@ -0,0 +1,45 @@
+# iPhone 11
+
+I've now spent one month using iPhone 11 as my daily driver.
+
+It is a big step up from my previous 250 EUR budget Android phone and my 5 year old iPhone 5S. This article should be seen in that light. I hadn't try a modern iOS device as my daily driver before so a lot of things that might be obvious to people using an iPhone X are not to me and I'm sure some things I point out were there in previous iPhones as well.
```

Comment:
I hadn't try -> I have not tried

Sentence is a run on, spilt it up

Reply:
Well now I have though